packages/ui/src/components/Icons.tsx

```diff
@@ -197,4 +200,92 @@ export const Icons: DDIcons = {
+          fill="white"
+        />
+      </g>
+    </svg>
+  ),
+  twitter_circle: (props: LucideProps) => (
+    <svg
+      width="80"
+      height="80"
+      viewBox="0 0 80 80"
+      {...props}
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <g id="Group 33">
+        <circle
+          id="Ellipse 4"
+          cx="40"
+          cy="40"
+          r="39"
+          fill="black"
+          fill-opacity="0.25"
+          stroke="#B7B7B7"
+          stroke-width="2"
+        />
+        <path
+          id="Vector"
+          d="M61 32.1729C59.6393 32.786 58.1777 33.2003 56.641 33.3876C58.2266 32.4228 59.4128 30.9043 59.9783 29.1154C58.4887 30.0152 56.8583 30.6486 55.1582 30.988C54.0148 29.7466 52.5005 28.9237 50.8502 28.6472C49.1999 28.3707 47.5061 28.656 46.0316 29.4589C44.5571 30.2617 43.3845 31.5371 42.6958 33.0872C42.0071 34.6372 41.8409 36.375 42.223 38.031C39.2046 37.8769 36.2518 37.079 33.5562 35.6892C30.8607 34.2995 28.4826 32.3488 26.5763 29.9639C25.9245 31.1073 25.5497 32.4331 25.5497 33.845C25.549 35.116 25.8567 36.3676 26.4457 37.4886C27.0347 38.6097 27.8867 39.5656 28.9261 40.2715C27.7207 40.2325 26.5419 39.9012 25.4878 39.3053V39.4048C25.4876 41.1874 26.094 42.9153 27.2039 44.2951C28.3139 45.6748 29.8591 46.6216 31.5773 46.9747C30.4591 47.2825 29.2867 47.3278 28.1488 47.1073C28.6336 48.6412 29.5779 49.9825 30.8495 50.9435C32.1211 51.9045 33.6564 52.437 35.2405 52.4665C32.5515 54.6133 29.2306 55.7777 25.812 55.7726C25.2065 55.7728 24.6014 55.7368 24 55.6649C27.47 57.9338 31.5094 59.138 35.6348 59.1333C49.5998 59.1333 57.2342 47.3708 57.2342 37.1692C57.2342 36.8378 57.226 36.5031 57.2114 36.1716C58.6963 35.0795 59.9781 33.7271 60.9967 32.1779L61 32.1729Z"
+          fill="white"
+        />
+      </g>
+    </svg>
+  ),
```

Comment:
Left a comment on the vercel preview branch but thought it might be easier if I leave it here as well :)

Shall we make  this the X logo > twitter Logo?

Reply:
I wish I could tag uma on this here lol. I think it's more on the design side.

Reply:
@uma-l

Reply:
created a github issue for this https://github.com/Developer-DAO/academy-turbo/issues/21 thanks Kemps!

@peteruche21 just add the icon and download it from x website when you can please, if we keep waiting on design I'm afraid we will end up not moving forward as we desire

regards !

Reply:
alright sure. will look into that

Reply:
Resolving conversation since we are going to work on this on later updates, thanks @kempsterrrr and @peteruche21  !